### PR TITLE
PREAPPS-6192 Adding certificate CRUD operations.

### PR DESCRIPTION
**Pull request by BhavinBathani**
Certificate management related graphql integrations.

### CODE REVIEW

File: src/batch-client/index.ts [hunk 1]
```diff
@@ -1218,6 +1220,14 @@ export class ZimbraBatchClient {
 			namespace: Namespace.Account
 		}).then(res => mapValuesDeep(res, coerceStringToBoolean));
 
+	public getSMimeCertInfo = () =>
+		this.jsonRequest({
+			name: 'GetSmimeCertificateInfo',
+			namespace: Namespace.Account
+		}).then(certificate => {
```
Comment (damini-vashishtha) [inline src/batch-client/index.ts:1227]: Implicit return.

File: src/batch-client/index.ts [hunk 2]
```diff
@@ -1630,6 +1640,18 @@ export class ZimbraBatchClient {
...
+	public saveSMimeCert = (upload: SaveSMimeCertInputUpload, password: string) =>
+		this.jsonRequest({
+			name: 'SaveSmimeCertificate',
+			body: {
+				upload,
+				password
+			},
+			namespace: Namespace.Account
+		}).then((certificate = []) => {
+			return normalize(SmimeCertInfoResponse)(certificate);
```
Comment (damini-vashishtha) [inline src/batch-client/index.ts:1652]: Implicit return.

File: src/batch-client/index.ts (hunk 1 above)
Comment (BhavinBathani) [reply src/batch-client/index.ts]: Fixed.

File: src/batch-client/index.ts (hunk 2 above)
Comment (BhavinBathani) [reply src/batch-client/index.ts]: Fixed.

**Review by damini-vashishtha — APPROVED** [review]

**Review by lavegupta — APPROVED** [review]

**Pull request merged**